### PR TITLE
Add Nextcloud variant

### PR DESCRIPTION
**Pull request by daniellandau**
This variant syncs with Nextcloud, based on the Dropbox code.

I'd be happy to provide a temporary Nextcloud account for testing if need be.

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -53,33 +54,40 @@ android {
...
+            applicationId 'nl.mpcjanssen.simpletask.nextcloud'
+            manifestPlaceholders = [providerFlavour: "nextcloud"]
         }
     }
 
     signingConfigs {
         release {
             keyAlias = "mpcjanssen.nl"
+            keyAlias 'my-key-alias'
+            keyPassword ')i8zu9\'=E\'Vh9}\'a'
```
Comment (mpcjanssen) [inline app/build.gradle]: was the password here intended?

**Comment by mpcjanssen** [comment]
I could add another flavor, but unless there is an easy way to add these additional backends as plugins instead of as full applications, I am a bit hesitant to merge this (it increases the full build time).

**Comment by mpcjanssen** [comment]
I could add it in a separate branch. But wont do that before you check the key config

File: app/build.gradle (hunk 1 above)
Comment (daniellandau) [inline app/build.gradle]: Nope. I'll get rid of that (it's not a key used for anything)

**Comment by mpcjanssen** [comment]
Added as nextcloud branch. Please don't use your signing key anymore.

**Pull request closed without merging**